lib/mongodb.js

```diff
@@ -765,7 +765,6 @@ MongoDB.prototype.parseUpdateData = function(modelName, data, options) {
       '$pop',
       '$pullAll',
       '$pull',
-      '$pushAll',
```

Comment:
any reason deleting this?

Reply:
`pushAll` is deprecated and being removed from MongoDB 3.6: https://docs.mongodb.com/manual/release-notes/3.6-compatibility/index.html#remove-pushall-update-operator